### PR TITLE
fix(helm): only default bucket names when using minio

### PR DESCRIPTION
**Pull request by trevorwhitney**
**What this PR does / why we need it**:

Only default bucket names in helm when using minio.

### CODE REVIEW

File: production/helm/loki/values.yaml [hunk 1]
```diff
@@ -311,10 +311,6 @@ loki:
     compactor_address: '{{ include "loki.compactorAddress" . }}'
   # -- Storage config. Providing this will automatically populate all necessary storage configs in the templated config.
   storage:
-    bucketNames:
-      chunks: chunks
-      ruler: ruler
-      admin: admin
```
Comment (slim-bean) [inline production/helm/loki/values.yaml:317]: instead of deleting this, maybe comment it out with FIXME for the names?

**Review by slim-bean — APPROVED** [review]
LGTM

**Comment by trevorwhitney** [comment]
@luong-komorebi would you mind sharing a bit of information about your install? I think the goal here is to _intentionally_ break any installation using the default buckets with a cloud provider as you likely don't own those buckets and don't mean to send data there? Was your installation using the defaults, and if so, were you only using it to query recent data?

**Comment by Farfaday** [comment]
Hi @trevorwhitney 

Commenting out `loki.storage.bucketNames` in values.yaml currently breaks helm lint, as the keys are not defined anymore but are called 9 times in the _helpers.tpl file.

Helm lint create errors of type  `nil pointer evaluating interface`.

I would propose adding a default value, something like
```
-  bucket_name: {{ $.Values.loki.storage.bucketNames.ruler }}
+  bucket_name: {{ $.Values.loki.storage.bucketNames.ruler | default "ruler" }}
```
What do you think? I can open an MR if you think that makes sense.

Thank you!

**Comment by thiagowfx** [comment]
@Farfaday Do you plan to open a PR? :)

**Pull request merged**